pyproject.toml

```diff
@@ -1,3 +1,42 @@
 [build-system]
 requires = ["setuptools>=41.2", "setuptools_scm", "wheel"]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "pyobis"
+dynamic = ["version"]
+description = "Python client for OBIS"
+readme = "README.md"
+requires-python = ">=3.7"
```

Comment:
Maybe take this opportunity to bump to Python >=3.10 to keep it above the EoL line.